### DISCUSSION TITLE
1000+ active installs in Google Play!

### DISCUSSION THREAD

**Discussion by syt0r** [syt0r/Kanji-Dojo#33] [Announcements]
Seems like the algorithm finally noticed Kanji Dojo. Almost twice the users in a month, hopefully this trend will keep going!
![image](https://user-images.githubusercontent.com/7478653/229306531-779cf95e-5562-4e1d-860c-899a2663a822.png)

Users count is cool but traffic is not quite healthy, people found an app using Google's recomendation, not by directly searching it. Hopefully new screenshot images and app descriptions (rewritten with the help of ChatGPT 😅) that I updated today will improve aquisition a little
![image](https://user-images.githubusercontent.com/7478653/229306610-8aac0a3f-ddad-4013-921d-d856cca2c25c.png)

**Comment by Audman** [comment]
Thank you for your hard work! 
This app helps me a lot
